src/pages/DevPage.js

```diff
@@ -174,7 +174,7 @@ function DevPage() {
               
               <Display
                 {...state}
-                conditions={{ fico: fico }}
+                conditions={{ fico: fico, pi4: true }}
```

Comment:
would it make sense to have this param be set via a checkbox instead so we can easily test non-Pi4 flows?